container/docker/handler.go

```diff
@@ -248,6 +251,7 @@ func newDockerContainerHandler(
 	handler.image = ctnr.Config.Image
 	handler.networkMode = ctnr.HostConfig.NetworkMode
 	handler.deviceID = ctnr.GraphDriver.Data["DeviceId"]
+	handler.restartCount = ctnr.RestartCount
```

Comment:
Yes, I was more making sure that we create a new dockerContainerHandler each time the container is restarted by docker.  Kubernetes doesn't use docker's restart policies, so I am not familiar with how we handle them.